pyproject.toml

```diff
@@ -22,7 +22,8 @@ precision = 1
 exclude_lines = [
     "pragma: no cover",
     "@(abc\\.)?abstractmethod",
-    "NotImplementedError\\(\\)"
+    "NotImplementedError\\(\\)",
+    "if TYPE_CHECKING:",
```

Comment:
I added this after discovering Ned [said](https://github.com/nedbat/coveragepy/issues/831#issuecomment-1020479593) it should be part of the default `exclude`.

I'll revert back to `# pragma: no cover` if you're not a fan.